modules/http4s-munit/src/main/scala/munit/config/package.scala

```diff
@@ -0,0 +1,17 @@
+package munit
+
+package object config {
```

Comment:
Why not creating this as `Config's` companion object?

```scala
package munit

final case class Config(repetitions: Option[Int], maxConcurrent: Option[Int])

object Config {

   lazy val default = Config(
     sys.props
       .get("http4s.munit.repetitions")
       .map(_.toIntOption)
       .flatten
       .orElse(sys.env.get("HTTP4S_MUNIT_REPETITIONS").map(_.toIntOption).flatten),
     sys.props
       .get("http4s.munit.maxConcurrent")
       .map(_.toIntOption)
       .flatten
       .orElse(sys.env.get("HTTP4S_MAX_CONCURRENT").map(_.toIntOption).flatten)
   )

 }
```

Reply:
See [`b976eac` (#42)](https://github.com/alejandrohdezma/http4s-munit/pull/42/commits/b976eacb063bb9e289c3a11cd6c8c8d2e49add84)